docs/usage/management-api-walkthrough/04_catalog.md

```diff
@@ -42,6 +38,8 @@ Content-Type: application/json
 The request body is lean. Mandatory properties are:
 - `counterPartyAddress` (formerly `providerUrl`): This property points to the DSP-endpoint of the Data Provider's Control
   Plane. Usually this ends on `/api/v1/dsp`.
+- `counterPartyId`: must be the provider BPN. This in not a mandatory property for EDC but it's required for TractusX-EDC.
+  If omitted the catalog request will fail.
```

Comment:
just curious: why does tractusx-edc require this additional bit of info in 0.7.x?